### ISSUE TITLE
Document autowiring of tasks

### ISSUE THREAD

**Issue by NicoHaase** [rewieer/TaskSchedulerBundle#7]
Just as some extension to your documentation: using `_instanceof`, as introduced at https://symfony.com/blog/new-in-symfony-3-3-simpler-service-configuration#interface-based-service-configuration, you can simplify the configuration. This is what I've used in a project to avoid configuring all tasks by hand:

    _instanceof:
        Rewieer\TaskSchedulerBundle\Task\TaskInterface:
            tags: ['ts.task']

**Comment by rewieer** [comment]
Thanks for the tip ! Can you submit a PR with this documentation in the README file ? Would be much appreciated.

**Comment by otakupahp** [comment]
@NicoHaase I added this line to my service.yaml, but I do not know if it is ok.

This is my class declaration:

```
namespace App\Tasks;

use App\Repository\DruckflaechenRepository;
use Rewieer\TaskSchedulerBundle\Task\AbstractScheduledTask;
use Rewieer\TaskSchedulerBundle\Task\Schedule;

class UpdateDFsize extends AbstractScheduledTask {

    /**
     * Set the Cron Schedule
     *
     * @param Schedule $schedule
     */
    protected function initialize(Schedule $schedule) {
        $schedule
            ->everyMinutes(5); // Perform the task every 5 minutes
    }

    /**
     * Run Update DFsize cron
     *
     * @return string|null  result of the cron or null if fails
     */
    public function run() : ?string {
        return "some string";
    }
}
```

**Comment by NicoHaase** [comment]
Sorry for not having reacted earlier, but I still do not get your problem. Is there something not working yet?

**Comment by otakupahp** [comment]
@NicoHaase thankyou for your reply.

Nothing works XD

Instead of adding a service.xml I edited my service.yaml adding this:

> _instanceof:
>         Rewieer\TaskSchedulerBundle\Task\TaskInterface:
>             tags: ['ts.task']

as you suggested in another comment, but when I run it in console, it does nothing.

Could you help me?

**Comment by NicoHaase** [comment]
Can you move this conversation to another forum, like stackoverflow.com? Currently, I cannot guess what the problem is, but this issue tracker is not the proper place for support questions

**Comment by otakupahp** [comment]
Good Idea, I will share the link here

**Comment by otakupahp** [comment]
@NicoHaase ,

Here is the Stackoverflow link

https://stackoverflow.com/questions/56973229/i-can-not-make-rewieer-taskschedulerbundle-for-symfony-work

**Comment by NicoHaase** [comment]
This is now handled in https://github.com/symfony/recipes-contrib/pull/895 - @rewieer could you check it and maybe help to get it included by writing a comment there?

**Comment by NicoHaase** [comment]
This was fixed in v0.5.0
